recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ about:
   summary: Labels and compares human languages in a standardized way
   license: MIT
   license_family: MIT
-  license_file: LICENSE
+  license_file: LICENSE.txt
```

Comment:
why is this `.txt` again? Isn#T the file you added called only `LICENSE` ?

Reply:
Hi @alexlang74 , This License file points to source code license and not to openCE one

Reply:
Yes, the license_file name here has to match the file name in the source code - https://github.com/rspeer/langcodes/blob/master/LICENSE.txt